data_model/src/asset.rs

```diff
@@ -387,6 +388,12 @@ impl fmt::Display for AssetId {
     }
 }
 
+impl fmt::Debug for AssetId {
```

Comment:
I know this is kind-of inconsistent, but I didn't really want to abuse proc-macros too much.